deployments/helm-chart/README.md

```diff
@@ -8,140 +8,159 @@ This chart deploys the NGINX Ingress Controller in your Kubernetes cluster.
 
+## CRDs
 
-## Getting the Chart Sources
-
-This step is required if you're installing the chart using its sources. Additionally, the step is also required for managing the custom resource definitions (CRDs), which the Ingress Controller requires by default, or for upgrading/deleting the CRDs.
-
-1. Clone the Ingress Controller repo:
-    ```console
-    $ git clone https://github.com/nginxinc/kubernetes-ingress --branch v3.0.2
-    ```
-    **Note**: If you want to use the experimental repository (`edge`), remove the `--branch` flag and value.
+By default, the Ingress Controller requires a number of custom resource definitions (CRDs) installed in the cluster. The Helm client will install those CRDs. If the CRDs are not installed, the Ingress Controller pods will not become `Ready`.
 
-2. Change your working directory to /deployments/helm-chart:
-    ```console
-    $ cd kubernetes-ingress/deployments/helm-chart
-    ```
+If you do not use the custom resources that require those CRDs (which corresponds to `controller.enableCustomResources` set to `false` and `controller.appprotect.enable` set to `false` and `controller.appprotectdos.enable` set to `false`), the installation of the CRDs can be skipped by specifying `--skip-crds` for the helm install command.
 
-## Adding the Helm Repository
+### Upgrading the CRDs
 
-This step is required if you're installing the chart via the helm repository.
+To upgrade the CRDs, you would need to pull the chart sources like described in [Pulling the Chart](#pulling-the-chart) and then run:
 
 ```console
-$ helm repo add nginx-stable https://helm.nginx.com/stable
-$ helm repo update
+$ kubectl apply -f crds/
 ```
+> **Note**
+>
+> The following warning is expected and can be ignored: `Warning: kubectl apply should be used on resource created by either kubectl create --save-config or kubectl apply`.
+>
+> Make sure to check the [release notes](https://www.github.com/nginxinc/kubernetes-ingress/releases) for a new release for any special upgrade procedures.
+
+### Uninstalling the CRDs
 
-**Note**: If you want to use the experimental repository, replace `stable` with `edge`.
+To remove the CRDs, you would need to pull the chart sources like described in [Pulling the Chart](#pulling-the-chart) and then run:
 
-## Installing the Chart
+```console
+$ kubectl delete -f crds/
+```
+> **Note**
+>
+> This command will delete all the corresponding custom resources in your cluster across all namespaces. Please ensure there are no custom resources that you want to keep and there are no other Ingress Controller releases running in the cluster.
 
-### Installing the CRDs
 
-By default, the Ingress Controller requires a number of custom resource definitions (CRDs) installed in the cluster. The Helm client will install those CRDs. If the CRDs are not installed, the Ingress Controller pods will not become `Ready`.
+## Managing the Chart via OCI Registry (edge version)
```

Comment:
I'm not 100% convinced about `Managing` but I couldn't come up with anything better (same for all the other sections)

Reply:
I know it's repetition with the other sections, but I feel `Installing` is better than `Managing`.

Reply:
Never mind, I've changed my mind as I kept going down the page 😆 Ignore me, managing is fine...